lib/public_key/src/public_key.erl

```diff
@@ -603,15 +605,16 @@ get_asn1_module('ECDSA-Sig-Value') -> 'PKIXAlgs-2009';
 get_asn1_module('RSAPrivateKey') -> 'PKCS-1';
 get_asn1_module('RSASSA-PSS-params') -> 'PKIX1-PSS-OAEP-Algorithms-2009';
 get_asn1_module('SubjectPublicKeyInfo') -> 'PKIX1Explicit-2009';
+get_asn1_module('SubjectAltName') -> 'PKIX1Implicit-2009';
 get_asn1_module('OTPTBSCertificate') -> 'OTP-PKIX';
 get_asn1_module('OTPCertificate') -> 'OTP-PKIX';
 get_asn1_module('CRLDistributionPoints') -> 'PKIX1Implicit-2009';
 get_asn1_module('CRLReason') ->  'PKIX1Implicit-2009';
 get_asn1_module('CRLNumber') ->  'PKIX1Implicit-2009';
 get_asn1_module('FreshestCRL') ->  'PKIX1Implicit-2009';
 get_asn1_module('IssuingDistributionPoint') ->  'PKIX1Implicit-2009';
-get_asn1_module('GeneralNames') -> 'PKIX1Implicit-2009'.
-
+get_asn1_module('GeneralNames') -> 'PKIX1Implicit-2009';
+get_asn1_module('Validity') -> 'OTP-PKIX'.
```

Comment:
Only OTP records should be taken from the OTP-PKIX is it available somewhere else?